src/main/java/net/pms/encoders/FFMpegVideo.java

```diff
@@ -431,10 +431,9 @@ public FFMpegVideo(PmsConfiguration configuration) {
 
 		// Give priority to the renderer's maximum bitrate setting over the user's setting
 		if (rendererMaxBitrates[0] > 0 && rendererMaxBitrates[0] < defaultMaxBitrates[0]) {
-			defaultMaxBitrates = rendererMaxBitrates;
-			LOGGER.trace("Using the video bitrate limit from the renderer config (" + rendererMaxBitrates[0] + "Mb/s) which is lower than the one from the program settings (" + defaultMaxBitrates[0] + ")");
+			LOGGER.trace("Using the video bitrate limit from the renderer config (" + rendererMaxBitrates[0] + " Mb/s) which is lower than the one from the program settings (" + defaultMaxBitrates[0] + " Mb/s)");
```

Comment:
@Nadahar It's why it's labelled as not ready ;-)
The complement was done by @valib in this PR: https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1152/files

Thanks for reviewing.